tests/numeric/test_PCATransformer.py

```diff
@@ -0,0 +1,416 @@
+import pytest
+import test_aide as ta
+import tests.test_data as d
+import pandas as pd
+
+import tubular
+from tubular.numeric import PCATransformer
+
+
+def created_numeric_df_1():
```

Comment:
would prefer this to be moved into tests.test_data as it may be reusable elsewhere

Reply:
Commit : [put df test example in test_data.py](https://github.com/lvgig/tubular/pull/57/commits/5fd8790623fa62501feb54badb7dc08909a20836)